README.md

```diff
@@ -176,11 +176,11 @@ jq.run('.', '/path/to/file.json', { output: 'string' }).then(console.log)
 |:----------------------------:|:---------------:|:-------:|
 | Read input stream into array | `true`, `false` | `false` |
 
-#### `slurp: 'true'`
+#### `slurp: true`
 
-Return the output as a **String**.
+Read input stream into array.
```

Comment:
I just got a little part from the description on the official jq docs ([here](https://stedolan.github.io/jq/manual/)):

> `--slurp/-s:`
> 
> Instead of running the filter for each JSON object in the input, read the entire input stream into a large array and run the filter just once.